roadmap.md

```diff
@@ -31,48 +31,61 @@ be refined if appropriate.
 * P0. Stable API for Remote execution including platform description
 * P0. List of feature to deprecate until version 1.0 are tracked in a publicly available document
 * P1. Bazel on Windows does not need to install MSYS
-* P2. Bazel can load workspace recursively
+* P1. Migration tool: Maven project to Bazel project
```

Comment:
Isn't this https://github.com/bazelbuild/migration-tooling#maven-to-bazel ? 
I understood it was on track. @kchodorow is the owner.